spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -5053,6 +5053,15 @@ for more of the supported options.
 
 
 
+[[boot-features-kafka-topics]]
+==== Creating Topics
+
+Spring Boot auto configuration will add a `KafkaAdmin` bean to the application context.
+This bean will automatically create a topic for each `NewTopic` bean in the application context.
+Topics can only be added with this technique, not modified; if a topic already exists, the `NewTopic` bean is ignored.
```

Comment:
I think it would be worthwhile to revisit this section with an example.

Reply:
OK; let me know your thoughts on the renamed property and I'll follow up with a polishing commit.